### PR TITLE
GPII-3739: Open USB folder.

### PR DESCRIPTION
**Pull request by stegru**
Back-end for the open usb button, which performs the following:

- If a single USB Drive is found, then that gets opened.
- If there's more than 1, then all drives that don't have a gpii token file are opened. (if all drives have a token then all are opened)

A "USB Drive" is a fixed or removal volume on the USB bus (which isn't a CD-ROM or Ramdisk).

Used by https://github.com/GPII/gpii-app/pull/92

### CODE REVIEW

**Comment by stegru** [comment]
oops, it looks like I've included a commit from something else - unrelated to this, so I've undone it.

File: gpii/node_modules/WindowsUtilities/WindowsUtilities.js [hunk 1]
```diff
@@ -1739,4 +1768,141 @@ gpii.windows.nativeExec = function (command, options, callback) {
...
+/**
+ * Used by the "Open USB" button. Gets the user's USB drives. That is, those without a token file.
+ * If all drives have a token file, then all are returned.
+ *
+ * @param {Array<String>} usbDrives [optional] The root paths to the usb drives to check, omit to check all USB drives
+ * on the system.
+ * @return {Array<String>} Array of paths to each drive.
+ */
+gpii.windows.getUserUsbDrives = function (usbDrives) {
+    usbDrives = usbDrives || gpii.windows.getUsbDrives();
```
Comment (amb26) [inline gpii/node_modules/WindowsUtilities/WindowsUtilities.js]: Although there are test cases, coverage report shows the entirety of these two functions as uncovered..... odd?

Comment (amb26) [reply gpii/node_modules/WindowsUtilities/WindowsUtilities.js]: My testing shows that the code is indeed hit, @the-t-in-rtf , can you comment?

Comment (amb26) [reply gpii/node_modules/WindowsUtilities/WindowsUtilities.js]: Ah sorry, no mystery - WindowsUtilitiesTests.js are not included in the unit tests!

**Comment by amb26** [comment]
Got a test failure in the vagrant box relating to order of the drives:
````
16:23:50.645:  jq: FAIL: Module "gpii.tests.windows.windowsUtilities" Test name "Testing getUserUsbDrives" - Message: getUserUsbDrives should resolve with the expected value - three drives, no token
16:23:50.645:  jq: Expected: [
    "A",
    "B",
    "C"
]
16:23:50.645:  jq: Actual: [
    "B",
    "C",
    "A"
]
````

File: gpii/node_modules/WindowsUtilities/WindowsUtilities.js (hunk 1 above)
Comment (the-t-in-rtf) [reply gpii/node_modules/WindowsUtilities/WindowsUtilities.js]: In addition, in anything that uses "node_modules" as a place to house code, there is always extra work to be done to keep nyc from ignoring it:

https://github.com/GPII/universal/blob/b4ef6b25c1586f7e79540e28ec04052d230bfb5a/.nycrc#L118

**Pull request merged**